time stamp

float32 comput_time
float32 max_comput_time
float32 mean_comput_time

float32 cpu_cores
float32 cpu_load
float32 cpu_max_load
float32 ram_usage_mb